ui/src/message_popup/author/edit_question_page.jsx

```diff
@@ -1,8 +1,8 @@
-/* @flow weak */
```

Comment:
Cool, sounds good to just work around for now and I can help take a look with you later if you like.